src/main/java/io/camunda/zeebe/bpmnassert/testengine/SinglePartitionDeploymentDistributor.java

```diff
@@ -0,0 +1,15 @@
+package io.camunda.zeebe.bpmnassert.testengine;
+
+import io.camunda.zeebe.engine.processing.deployment.distribute.DeploymentDistributor;
+import io.camunda.zeebe.util.sched.future.ActorFuture;
+import io.camunda.zeebe.util.sched.future.CompletableActorFuture;
+import org.agrona.DirectBuffer;
+
+public class SinglePartitionDeploymentDistributor implements DeploymentDistributor {
```

Comment:
:thought_balloon: We can also align on whether to use final for classes and whether or not to limit visibility to the minimum. In the DB folder must stuff is final and package private. On the other hand, it's also frustrating when you want to extend this library and you cannot because we put all the final keywords in and hid stuff.